### PR TITLE
Add additional .PHONY tags to appropriate make targets

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

**Comment by cap10morgan** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/fluree/server/66?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#66** <a href="https://app.graphite.dev/github/pr/fluree/server/66?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#65** <a href="https://app.graphite.dev/github/pr/fluree/server/65?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cap10morgan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,7 @@ RESOURCES := $(shell find resources)
 target/server-%.jar: $(SOURCES) $(RESOURCES) prepare
 	clojure -T:build uber
 
+.PHONY: uberjar
```
Comment (dpetran) [inline Makefile:7]: What does .PHONY signify in a Makefile?

Comment (cap10morgan) [reply Makefile:7]: That the target is not a filename that the rule will create (which is the default). Without this, if a file named `uberjar` were created and had a newer timestamp than the prerequisites, make would say "nothing to do for uberjar" when you ran `make uberjar`. Make calls these "phony targets."

Comment (dpetran) [reply Makefile:7]: Thanks, that makes sense.

**Review by dpetran — APPROVED** [review]
:+1:

**Pull request merged**